src/Infrastructure/Infrastructure.csproj

```diff
@@ -1,19 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This is not really in the scope of this pull request, but we should find a way to configure Visual Studio and Visual Studio Code such that they do not clash with each other.

We use a tab size of four for the C# code I think we should do this for XML and JSON too. I am fine with using a tab size of two though, provided that we enforce it everywhere.

Reply:
Doesn't editorconfig handle these things for both?

Reply:
Do do not have one yet. I created #16 to add one.